media_player/alexa.py

```diff
@@ -436,12 +443,23 @@ def _get_source_list(self):
             for devices in self._bluetooth_state['pairedDeviceList']:
                 sources.append(devices['friendlyName'])
         return ['Local Speaker'] + sources
+    
+    def _get_last_called(self):
+        last_device_serial = self.alexa_api.get_last_device_serial()
+        if self._device_serial_number == last_device_serial:
+            return True
+        return False
 
     @property
     def available(self):
         """Return the availability of the client."""
         return self._available
 
+    @property
+    def last_device_serial(self):
+        """Return the last activity of the client."""
+        return self._last_device_serial_number
```

Comment:
Where is this value set?

Reply:
I believe this is was used in my first round. I thought I had removed all of that code. Thanks for catching it.